sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_get_bounding_boxes_async.py

```diff
@@ -57,15 +60,14 @@ async def get_bounding_boxes(self):
             forms = await poller.result()
             for idx, form in enumerate(forms):
                 print("--------RECOGNIZING FORM #{}--------".format(idx))
-                print("Form has type {}".format(form.form_type))
+                print("Form has type ID: {}".format(form.form_type))
```

Comment:
Are we really calling this ID?

Reply:
I think this will eventually be a form type, like receipt, invoice, etc. but right now it's more of an ID value e.g. "0" or "form-xxxxxxxxxxxxxx". Thought I'd update current samples to reflect that, but don't feel super strongly about it.